docs/run-windows-cli.md

```diff
@@ -16,6 +16,9 @@ Run this from an existing React Native for Windows app to build and deploy.
 npx react-native run-windows
 ```
 ### Options
+
+> **Note:** Remote Debugging was officially marked as deprecated in RN 0.73 and will be removed in a later release.
```

Comment:
Is there a release note or discussions-and-proposals issue you can link to here?

Reply:
Updated!